warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__modification_date_present_rt.sql

```diff
@@ -10,30 +15,40 @@ daily_modification_date_status AS (
     SELECT dt AS date,
            base64_url,
            feed_type,
-           MAX(last_modified_string) AS max_last_modified_string
+           LOGICAL_OR(last_modified_string IS NOT NULL) AS has_last_modified_string
```

Comment:
no requested change but I'm realizing we check for `Last-Modified` in schedule but `last_modified` in RT and I don't remember why 🤔

Reply:
@owades do you remember why? 😅 Happy to change if needed. Can also do as a follow up.

Reply:
This isn't a great answer but this is just what returned results. See the [PR](https://github.com/cal-itp/data-infra/pull/2180#pullrequestreview-1260914045). Using Last-Modified for RT returned only null results.

Reply:
lol @atvaccaro is nothing if not consistent 😂 https://github.com/cal-itp/data-infra/pull/2180#discussion_r1080656680

Reply:
Ah I bet we call `make_name_bq_safe` in RT but not schedule

Reply:
Probably not ideal in situations where we treat things as JSON blobs rather than structs to be explicitly typed :/